samples/snippets/csharp/VS_Snippets_CLR/HowToSignXMLDocumentRSA/cs/sample.cs

```diff
@@ -70,7 +70,7 @@ public static void SignXml(XmlDocument xmlDoc, RSA Key)
 
         // Add the key to the SignedXml document.
         // <snippet6>
-        signedXml.SigningKey = rsaKey;
+        signedXml.SigningKey = Key;
```

Comment:
Parameter names should use camelCase: it should be `key`, not `Key`. Do you want to also make that change (both on this line and on lines 58, 63 and 64 above)?

Reply:
If you don't know how to change your Pull Request, that's fine. Somebody can make that change in a separate Pull Request.